src/amo/components/Card/index.js

```diff
@@ -63,7 +63,13 @@ export default class Card extends React.Component {
           'Card--no-footer': !footer,
         })}
       >
-        {header ? <header className="Card-header">{header}</header> : null}
+        {header ? (
+          <header className="Card-header">
+            <div className="Card-header-text">{header}</div>
+
+            {footer ? <div className="Card-text-wrapper">{footer}</div> : null}
```

Comment:
This `className` isn't really descriptive of what this is. Maybe something like `Card-footer-in-header` would be better?